orc8r/gateway/python/magma/common/stateless_agw.py

```diff
@@ -69,10 +69,7 @@ def disable_stateless_agw():
         logging.info("Nothing to disable, AGW is stateful")
     for service, config, _ in STATELESS_SERVICE_CONFIGS:
         cfg = load_override_config(service) or {}
-
-        # remove the stateless override
-        cfg.pop(config, None)
-
+        cfg[config] = not value
```

Comment:
where is this `value` coming from? i think this needs to be read from `STATELESS_SERVICE_CONFIGS`, similarly the way is being done on `config_stateless_agw.py` script

Reply:
Fixed in the latest commit